src/sentry/conf/server.py

```diff
@@ -3752,6 +3752,11 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 # For encrypting the access token for the GHE integration
 SEER_GHE_ENCRYPT_KEY: str | None = os.getenv("SEER_GHE_ENCRYPT_KEY")
 
+# Used to validate RPC requests from the Overwatch service
+# TODO: Update to list[str] for key rotation support like other RPC secrets
```

Comment:
what's preventing us from doing this now?

Reply:
To clarify, I meant automatically rotating. I agree with making it a list to help with a future rotation.